wondrous-app/components/Common/PodIconName.tsx

```diff
@@ -0,0 +1,47 @@
+import { Grid, Typography } from '@mui/material';
+import PodIcon from 'components/Icons/podIcon';
+import palette from 'theme/palette';
+
+interface PodIconNameProps {
+  color?: string;
+  name: string;
+  onClick?: (e: any) => void;
+  IconComponent?: () => JSX.Element;
+}
+
+const PodIconName = ({
+  color = palette.grey900,
+  name,
+  onClick = null,
+  IconComponent = () => (
```

Comment:
why do we pass this in as input? seems like we can just hard code this in?

Reply:
@terryli0095 I added parameters so we can reuse it. For example:

<img width="363" alt="image" src="https://user-images.githubusercontent.com/8164667/201804163-10294f4b-db3a-4d52-a593-883c85cd14b7.png">